spanner/benchmark/ycsb.py

```diff
@@ -0,0 +1,257 @@
+"""The YCSB client in Python.
+
+Usage:
+
+  # Set up instance and load data into database.
+
+  # Set up environment variables. You should use your own credentials and gcloud
+  # project.
+  $ export GOOGLE_APPLICATION_CREDENTIALS=/path/to/credentials.json
+  $ export GCLOUD_PROJECT=gcloud-project-name
+
+  # Run the benchmark.
+  $ python spanner/benchmark/ycsb.py run -P pkb/workloada -p table=usertable \
+    -p cloudspanner.instance=ycsb-542756a4 -p recordcount=5000 \
+    -p operationcount=100 -p cloudspanner.database=ycsb -p num_worker=1
+
+  # To make a package so it can work with PerfKitBenchmarker.
+  $ cd spanner; tar -cvzf ycsb-python.0.0.5.tar.gz benchmark/*
+
+"""
+
+import numpy
+import optparse
+import random
+import string
+import threading
+import timeit
+
+from google.cloud import spanner
+
+
+KEYS = []
+OPERATIONS = ['readproportion', 'updateproportion', 'scanproportion',
+              'insertproportion']
+NUM_FIELD = 10
+
+
+def parse_options():
+  """Parses options."""
```

Comment:
Is this file using 2 spaces? Our public style guide is 4.

Reply:
Done in my next commit.